lite2/provider/http/http_test.go

```diff
@@ -57,26 +57,26 @@ func TestProvider(t *testing.T) {
 	require.Nil(t, err)
 
 	p := litehttp.NewWithClient(chainID, c)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	require.NotNil(t, p)
 
 	// let it produce some blocks
-	err = rpcclient.WaitForHeight(c, 6, nil)
-	require.Nil(t, err)
+	err = rpcclient.WaitForHeight(c, 10, nil)
+	require.NoError(t, err)
 
 	// let's get the highest block
 	sh, err := p.SignedHeader(0)
 
-	require.Nil(t, err, "%+v", err)
-	assert.True(t, sh.Height < 5000)
+	require.NoError(t, err)
+	assert.True(t, sh.Height < 1000)
 
 	// let's check this is valid somehow
 	assert.Nil(t, sh.ValidateBasic(chainID))
 
 	// historical queries now work :)
 	lower := sh.Height - 3
 	sh, err = p.SignedHeader(lower)
-	assert.Nil(t, err, "%+v", err)
+	assert.NoError(t, err)
```

Comment:
This should be `require` as well, otherwise the below will throw a nil panic on error.